### PR TITLE
schema migrated to version 44

### PR DESCRIPTION
**Pull request by subham-71**
## Description of the change

schema migrated to version 44. pb compile ran successfully.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: logs/logfile.log
Comment (dpatchigolla) [inline logs/logfile.log:1]: Please add logfile.log to .gitignore

File: models/ml-features.yaml
Comment (dpatchigolla) [inline models/ml-features.yaml:1]: Can you change this file name to profiles.yaml? That's a long overdue change.

File: models/ml-features.yaml [hunk 1]
```diff
@@ -55,250 +55,233 @@ models:
...
             from: inputs/stripeCharges
             name: last_purchase_time
-            select: max(created)
+            select: max({{stripeCharges.Var('created')}})
             where: status = 'succeeded'
         - entity_var:
             name: days_since_last_purchase
-            select: "{{ macro_datediff('last_purchase_time') }}"
+            select: "{{ macro_datediff('{{user.Var(\"last_purchase_time\")}}') }}"
             dependencies:
```
Comment (dpatchigolla) [inline models/ml-features.yaml:124]: @yajuaryarudderstack we don't need dependencies if we have the user.Var right?

**Review by dpatchigolla — COMMENTED** [review]
Added a couple of comments.

File: models/ml-features.yaml (hunk 1 above)
Comment (yajuaryarudderstack) [reply models/ml-features.yaml:124]: @dpatchigolla Yes, if we are using <entityName>.Var then no need to use dependencies, it's redundant.

**Comment by subham-71** [comment]
I have made the changes as per the comments.
- Changed the file name
- Removed the dependencies
- Updated gitignore

**Review by dpatchigolla — APPROVED** [review]

**Pull request merged**